src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -325,19 +320,14 @@
                                                       (re-frame/dispatch [:chat.ui/set-chat-input-text text])))}]
        [send-button inp-ref disconnected?]])))
 
-(defn not-joined-group-chat? [chat current-public-key]
-  (and (models.chat/group-chat? chat)
-       (models.group-chats/invited? current-public-key chat)
-       (not (models.group-chats/joined? current-public-key chat))))
-
 (views/defview chat-view []
-  (views/letsubs [{:keys [input-text chat-id] :as current-chat} [:chats/current-chat]
+  (views/letsubs [{:keys [input-text chat-id inviter-name] :as current-chat} [:chats/current-chat]
                   current-public-key [:account/public-key]]
 
     [react/view {:style styles/chat-view}
      [toolbar-chat-view current-chat]
      [react/view {:style styles/separator}]
-     (if (not-joined-group-chat? current-chat current-public-key)
+     (if inviter-name
```

Comment:
it's not very clear that inviter-name is available only if the user has not joined the chat, it'd be nice to find a better name/interface, but can't think of anything (other that splitting the two), not sure if you have any idea, otherwise is ok

Reply:
`pending-invite-inviter-name` ?

Reply:
sounds good!